protocol-units/bridge/integration-tests/src/lib.rs

```diff
@@ -49,29 +52,42 @@ pub struct MovementToEthCallArgs {
 }
 
 impl Default for EthToMovementCallArgs {
-	fn default() -> Self {
-		Self {
-			initiator: b"0x123".to_vec(),
-			recipient: MovementAddress(AccountAddress::new(*b"0x00000000000000000000000000face")),
-			bridge_transfer_id: MovementHash(*b"00000000000000000000000transfer1"),
-			hash_lock: MovementHash(*keccak256(b"secret")),
-			time_lock: 3600,
-			amount: 100,
-		}
-	}
+        fn default() -> Self {
+                // Generate 6 random alphanumeric characters
+                let random_suffix: String = thread_rng()
+                    .sample_iter(&Alphanumeric)
+                    .take(6)
+                    .map(char::from)
+                    .collect();
+
+                // Construct the bridge_transfer_id with the random suffix
+                let mut bridge_transfer_id = b"00000000000000000000000tra".to_vec();
+                bridge_transfer_id.extend_from_slice(random_suffix.as_bytes());
+
+                Self {
+                        initiator: b"32Be343B94f860124dC4fEe278FDCBD38C102D88".to_vec(),
```

Comment:
Where does this address come from?

Reply:
It's the dummy valid EIP-55 address used in the framework modules.

Reply:
I'll add a comment.